modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,119 @@
+|`integer`
+|`--autoscaler-pod-priority-threshold -10`
+
+|`--autoscaler-gpu-limit _stringArray_`
+|Minimum and maximum number of different GPUs in cluster. Cluster autoscaler does not scale the cluster beyond these numbers. The format should be a comma-separated list of "<gpu_type>,<min>,<max>".
+|`array`
+|`--autoscaler-gpu-limit nvidia.com/gpu,0,10  --autoscaler-gpu-limit amd.com/gpu,1,5`
+
+|`--autoscaler-ignore-daemonsets-utilization`
+|If set, the cluster-autoscaler ignores daemon set pods when calculating resource utilization for scaling down.
+|`boolean`
+|Add it to set to true, omit the option to set to false.
+
+|`--autoscaler-max-node-provision-time _string_`
+|Maximum time that the cluster-autoscaler waits for node to be provisioned. Replace _string_ in the command with an integer and time unit (ns,us,µs,ms,s,m,h).
+|`string`
+|`--autoscaler-max-node-provision-time 35m`
+
+|`--autoscaler-balancing-ignored-labels _strings_`
+|A comma-separated list of label keys that the cluster autoscaler should ignore when considering node group similarity. Replace _strings_ in the command with the relevant labels.
+|`string`
+|`--autoscaler-balancing-ignored-labels topology.ebs.csi.aws.com/zone,alpha.eksctl.io/instance-id`
+
+|`--autoscaler-max-nodes-total _int_`
+|Maximum amount of nodes in the cluster, including the autoscaled nodes. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-nodes-total 180`
+
+|`--autoscaler-min-cores _int_`
+|Minimum number of cores to deploy in the cluster. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-min-cores 0`
+
+|`--autoscaler-max-cores _int_`
+|Maximum number of cores to deploy in the cluster. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-cores 100`
+
+|`--autoscaler-min-memory _int_`
+|Minimum amount of memory, in GiB, in the cluster. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-min-memory 0`
+
+|`--autoscaler-max-memory _int_`
+|Maximum amount of memory, in GiB, in the cluster. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-memory 4096`
+
+|`--autoscaler-scale-down-enabled`
+|If set, the cluster autoscaler should scale down the cluster.
+|`boolean`
+|Add it to set to true, omit the option to set to false.
+
+|`--autoscaler-scale-down-unneeded-time _string_`
+|How long a node should be unneeded before it is eligible for scale down. Replace _string_ in the command with an integer and time unit (ns,us,µs,ms,s,m,h).
+|`string`
+|`--autoscaler-scale-down-unneeded-time 1h`
+
+|`--autoscaler-scale-down-utilization-threshold _float_`
+|Node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down. Value should be between 0 and 1.
```

Comment:
```suggestion
|Node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down. Value must be between 0 and 1.
```
Cluster capacity? This behavior appears in the table below as well, but doesn't have the value requirement. Should it?